Example/LMGRemoteData.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this supposed to be committed?

Reply:
I dont think so, I fixed it.

Reply:
Can you add the `xcshareddata/` folder to the `.gitignore` so these don't get accidentally added again please?